### PR TITLE
Add screen.bounds #40

### LINKED ISSUE #40: Add screen.bounds
Originally reported by: **Daniel Pope (Bitbucket: [lordmauve](https://bitbucket.org/lordmauve), GitHub: [lordmauve](https://github.com/lordmauve))**

----------------------------------------

Add a method or property to get the Rect for the screen itself.

This would make it easier to test for collision with the edge of the screen.

----------------------------------------
- Bitbucket: https://bitbucket.org/lordmauve/pgzero/issue/40

### PR DESCRIPTION
**Pull request by alisdt**
Add a bounds method for Screen. Resolves #40, but affected by #94 if the user then uses it to (e.g.) check an Actor is on the screen with .contains()

### CODE REVIEW

File: pgzero/screen.py [hunk 1]
```diff
@@ -72,6 +72,10 @@ def __init__(self, surface):
         self.surface = surface
         self.width, self.height = surface.get_size()
 
+    def bounds(self):
+        """Returns a Rect representing the bounds of the screen"""
```
Comment (lordmauve) [inline pgzero/screen.py]: Nitpicking, but this docstring is not PEP-257 compliant - it should be in the imperative mood ("Return" not "Returns") and it should end with a .

**Review by lordmauve — COMMENTED** [review]
Should we just make screen.bounds() return a ZRect given this falls foul of #94?

**Comment by alisdt** [comment]
Fixed this, and the docstring.

**Comment by lordmauve** [comment]
Thank you!

**Pull request merged**